src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Output/StandardColumnDataGenerator.cs

```diff
@@ -44,9 +44,9 @@ internal class StandardColumnDataGenerator : IStandardColumnDataGenerator
             switch (column)
             {
                 case StandardColumn.Message:
-                    return new KeyValuePair<string, object>(_columnOptions.Message.ColumnName, logEvent.RenderMessage(_formatProvider));
+                    return new KeyValuePair<string, object>(_columnOptions.Message.ColumnName, RenderLogMessage(logEvent));
                 case StandardColumn.MessageTemplate:
-                    return new KeyValuePair<string, object>(_columnOptions.MessageTemplate.ColumnName, logEvent.MessageTemplate.Text);
+                    return new KeyValuePair<string, object>(_columnOptions.MessageTemplate.ColumnName, TrimMessageTemplate(logEvent));
```

Comment:
Does `Exception` and `Properties` also need to be checked, or are they assumed to be `varchar(max)`?

Reply:
For the specific use case in which I/we use the sink, only a truncate on the `Message` and `MessageTemplate` is a necessity. Reason: there are indexes created on those columns of the log table.

I don't mind truncating these fields as well. Although I've to admit that it probably makes more sense for the `Exception` field than it does for `Properties`. The exception message is just a string while the properties are really a structure. In our case, the `Properties` is mapped on a `nvarchar(max)` field.

Shall I only truncate the `Exception` field or also the `Properties` (and `LogEvent`) fields?

Reply:
I only added an extra call to truncate the exception message (alongside log message and message template), as I believe it will be very unlikely that a use case to truncate the properties collection really is necessary.